extensions/mssql/build/postinstall.js

```diff
@@ -15,15 +15,6 @@
 	async function installService() {
 		const absoluteConfigPath = require.resolve('../config.json');
 		const config = require(absoluteConfigPath);
-		let runtime = (await platform.PlatformInformation.getCurrent()).runtimeId;
```

Comment:
this is removed because I've added the runtime override support in the ads-service-downloader package. we just need to specify the AZDATA_RUNTIME in the yml file.